sungc/deglint.py

```diff
@@ -546,27 +550,45 @@ def nir_subtraction(
                 )
 
                 # ------------------------------ #
-                #       Resample NIR band        #
+                #        Resample NIR band       #
+                #               and              #
+                #       Sunglint Correction      #
                 # ------------------------------ #
+                deglint_band = np.array(vis_im, order="K", copy=True)
+
                 if (nir_nRows != dsVIS.height) or (nir_nCols != dsVIS.width):
                     # resample the NIR band to match the VIS band
-                    nir_im = rio_funcs.resample_file_to_ds(
+                    nir_im_res = rio_funcs.resample_file_to_ds(
                         nir_bandPath, dsVIS, Resampling.bilinear
                     )
 
-            # ------------------------------ #
-            #       Sunglint Correction      #
-            # ------------------------------ #
-            # copy band
-            deglint_band = np.array(vis_im, order="K", copy=True)
+                    waterIx = np.where(
```

Comment:
Is this a **TAB**?

Reply:
Definitely not a TAB, plus those lines have been removed.